apps/jobs/views.py

```diff
@@ -289,11 +297,63 @@ class UserViewSets(viewsets.ModelViewSet):
     queryset = User.objects.all()
     serializer_class = UserSerializer
 
-    def create(self, request, *args, **kwargs):
+    def update(self, request, *args, **kwargs):
         """
         Overriding the create method (used in POST request),
         This method creates a new user profile in the database.
         """
+        common_response_parameters = {
+            "status": status.HTTP_400_BAD_REQUEST,
+            "content_type": "application/json",
+        }
+
+        if request.headers and "AccessToken" in request.headers:
+            # decode the "user_id" from AccessToken
+            try:
+                payload = jwt.decode(
+                    request.headers["AccessToken"], options={"verify_signature": False}
+                )
+            except jwt.exceptions.DecodeError:
+                return Response(
+                    {"error": "AccessToken is not valid"}, **common_response_parameters
+                )
+            except Exception as err:
+                common_response_parameters[
+                    "status"
+                ] = status.HTTP_500_INTERNAL_SERVER_ERROR
+                return Response(
+                    {"error": "Something went wrong"}, **common_response_parameters
+                )
```

Comment:
keep different response under constant/response.py

Reply:
Created a function in `constants/response.py` where you need to pass only the message and status code, and it'll return the Response object with message, status and content_type